src/app/components/Byline/index.styles.tsx

```diff
@@ -2,12 +2,29 @@ import { css, Theme } from '@emotion/react';
 import pixelsToRem from '../../utilities/pixelsToRem';
 
 export default {
-  bylineSection: ({ spacings, mq }: Theme) =>
+  bylineContainer: ({ spacings, mq }: Theme) =>
     css({
+      display: 'grid',
```

Comment:
Grid might not work here, specifically for opera mini https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/Styles.mdx#css-grid-can-only-be-used-for-page-layout

Reply:
I think that might be accpetable in this case given the fall back behaviour : `This fallback behaviour in most cases will provide an acceptable experience for a mobile page layout. Given Opera Mini is a mobile only browser, we can assume that a single column page layout will be acceptable` , on the lowest break point we want a single column view anyway.